tests/e2e/e2e_test.go

```diff
@@ -25,6 +25,15 @@ import (
 	"github.com/osmosis-labs/osmosis/v14/tests/e2e/initialization"
 )
 
+func (s *IntegrationTestSuite) TestConcentratedLiquidity() {
+	chainA := s.configurer.GetChainConfig(0)
+	node, err := chainA.GetDefaultNode()
+	s.Require().NoError(err)
+
+	concentratedPool := node.CreateConcentratedPool(initialization.ValidatorWalletName)
+	fmt.Println("CONCENTRATED POOL: ", concentratedPool)
```

Comment:
Once you refactor to add the requested CL pools query, please call it here. When the pool is returned, make assertions about the contents of the pool.